File: LLM-Translations/GPT-4/codenet/Python/C/s000375264.c
#include <stdio.h>

int main() {
    int X, t, answer;
    scanf("%d %d", &X, &t);

    if (X > t) {
        answer = X - t;
    } else if (X <= t) {
        answer = 0;
    }

    printf("%d\n", answer);
    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s000375264.go
package main

import (
	"fmt"
)

func main() {
	var X, t int
	fmt.Scan(&X, &t)

	var answer int
	if X > t {
		answer = X - t
	} else if X <= t {
		answer = 0
	}

	fmt.Println(answer)
}
// 
